src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead, style elements using `.class`. Here, it would be better to use a class for the `<ul>` element, such as `.nav__list`, and apply the `list-style: none;` to that class instead of the `ul` tag.